src/django_cognito_jwt/backend.py

```diff
@@ -2,8 +2,8 @@
 
 from django.apps import apps as django_apps
 from django.conf import settings
-from django.utils.encoding import smart_text
```

Comment:
At line 41 `smart_text` is still used, can you replace that one for `force_str`?